### DISCUSSION TITLE
The problem is that hubs (admin) directly contact 8.8.8.8

### DISCUSSION THREAD

**Discussion by kouzzy** [mozilla/hubs#5495] [Hubs Cloud Questions] [answered]
1. Download scene and avatar data by ".pack etc." in "admin menu-import content"
2. IP query occurs directly from the hubs (admin) system to [8.8.8.8] (dns.google)
3. I'm trying to find "8.8.8.8" inside the program that makes up admin, but I don't know where it is.

Reason for problem:
I'm in trouble because my company has strict rules and can't add [8.8.8.8] to the whitelist.

I'm sorry if it has been solved by a known problem.

**Comment by kouzzy** [comment, marked as answer]
Self-solving

The solution is to change "8.8.8.8" in "reticulum / deps / dns / lib / dns.ex" and recompile.

However, I think it is a problem to set the DNS contact to reticlum with static "8.8.8.8", so I will raise the problem with reticulum after this.

**Reply by pattersonbl2** [reply]
Thank you for this finding. if you have created an issue, can you please link it here?

**Reply by kouzzy** [reply]
Thank you for your infomation .
The following discussion : [https://github.com/tungd/elixir-dns/issues/50#issue-1268198067]
